add&remove.js

```diff
@@ -0,0 +1,68 @@
+export default class AddRemove {
+  static Add(data) {
+    this.addNew = document.querySelector('.input1');
+    this.addNewBtn = document.querySelector('.input2');
+    this.addNewBtn.addEventListener('click', (e) => {
+      if (this.addNew.value === '') {
+        e.preventDefault();
+      } else {
+        data.push({
+          description: this.addNew.value,
+          completed: false,
+          index: null,
+        });
+        localStorage.setItem('Task', JSON.stringify(data));
+      }
+    });
+    this.insert = document.getElementById('ul-list');
+    this.allCompleted = document.querySelector('.clear');
+    this.allCompleted.addEventListener('click', () => {
+      this.checks = document.querySelectorAll('.label-true');
+      this.checks.forEach((e) => {
+        data.splice(e, 1);
+        this.insert.removeChild(e);
+        localStorage.setItem('Task', JSON.stringify(data));
```

Comment:
Instead of repeating a line of code put this in a separate method and call it. 
**For example:** 
const setStorage = () => {
localStorage.setItem('Task', JSON.stringify(data));
}